src/configuration/sales/multishipping-settings.md

```diff
@@ -13,3 +13,7 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 |--- |--- |--- |
 |Allow Shipping to Multiple Addresses|Website|Determines if you allow single orders to be shipping to multiple addresses. Options: Yes / No|
 |Maximum Qty Allowed for Shipping to Multiple Addresses|Website|Sets a limit for the number of units  of a product that can be shipping to multiple addresses.|
+
+{:.bs-callout-info}
+<span class="b2b-only"></span>  
```

Comment:
I was struggling to find a way to highlight that this callout is about a B2B related feature. I wasn't able to make `{:.b2b-only}` work next to `{:.bs-callout-info}`, but I found some usages of HTML code like here `src/system/index-management.md:36` that was added relatively recently. Please correct me if there is a better way to do this while keeping it look good :) Thanks.